docs/guide/recovery.md

```diff
@@ -9,7 +9,7 @@ nav_order: 2
 {: .pb-2 }
 
 {: .important }
-> This guide is specific to devices in which the install method is marked as **"Custom recovery"**.
+> This guide is specific to the devices in which the install method is marked as **"Custom recovery"**.
```

Comment:
This should also be matching in:
https://github.com/BlackMesa123/UN1CA/blob/ea95478e6f0d093a98e6fea51cbdda3fe721a40e/docs/guide/download.md?plain=1#L12

Reply:
Addressed in 1cf689150b92a7a296f54b6991cc8bf8b51f2cda.